tests/gocase/unit/type/zset/zset_test.go

```diff
@@ -235,6 +235,62 @@ func basicTests(t *testing.T, rdb *redis.Client, ctx context.Context, encoding s
 		require.Equal(t, int64(3), rdb.ZRem(ctx, "ztmp", []string{"a", "b", "c", "d", "e", "f", "g"}).Val())
 	})
 
+	t.Run(fmt.Sprintf("ZPOPMIN basics - %s", encoding), func(t *testing.T) {
+		rdb.Del(ctx, "ztmp")
+		rdb.ZAdd(ctx, "ztmp", redis.Z{Score: 10, Member: "a"}, redis.Z{Score: 20, Member: "b"}, redis.Z{Score: 30, Member: "c"})
+		require.Equal(t, int64(3), rdb.ZCard(ctx, "ztmp").Val())
+		require.Equal(t, []redis.Z{{Score: 10, Member: "a"}}, rdb.ZPopMin(ctx, "ztmp").Val())
+		require.Equal(t, int64(2), rdb.ZCard(ctx, "ztmp").Val())
+		require.Equal(t, []redis.Z{{Score: 20, Member: "b"}}, rdb.ZPopMin(ctx, "ztmp").Val())
+		require.Equal(t, int64(1), rdb.ZCard(ctx, "ztmp").Val())
+		require.Equal(t, []redis.Z{{Score: 30, Member: "c"}}, rdb.ZPopMin(ctx, "ztmp").Val())
+		require.Equal(t, int64(0), rdb.ZCard(ctx, "ztmp").Val())
+		require.Equal(t, []redis.Z{}, rdb.ZPopMin(ctx, "ztmp").Val())
+		rdb.ZAdd(ctx, "ztmp", redis.Z{Score: 10, Member: "a"}, redis.Z{Score: 20, Member: "b"}, redis.Z{Score: 30, Member: "c"})
+		require.Equal(t, int64(3), rdb.ZCard(ctx, "ztmp").Val())
+		require.Equal(t, []redis.Z{{Score: 10, Member: "a"}, {Score: 20, Member: "b"}}, rdb.ZPopMin(ctx, "ztmp", 2).Val())
+		require.Equal(t, int64(1), rdb.ZCard(ctx, "ztmp").Val())
```

Comment:
You can use `require.EqualValues` (a handy thing) here and in other similar places to get rid of explicit conversions like `int64(1)`.

Reply:
Fixed.